### DISCUSSION TITLE
Making placement of armaments (a.k.a. buildings/fortifications) matter.

### DISCUSSION THREAD

**Discussion by sum1els-com** [jacobsalmela/ascii-planets#38] [Ideas]
As the rules stand today, armaments are allowed to touch.  There is a penalty for construction of an armament on a mountain or in a nebula, but there is so much space that no player should ever need to make a decision about placement.  Even on the new smaller (and rounder) planet map there is no shortage of "free" real estate.

One simple change to the rules would be to disallow armament touching.  On the current planet itself, however I can fit 13 armaments without straying into the mountains and without touching.  Since there are only 3 land-based armaments, it's easy to fit 4 of each type, which should be plenty for an entire game.  So, simply disallowing armament touching isn't enough to put some pressure on the players.  In addition to a no-touch rule, I would consider adding a few more mountain ranges.

In space the story is even worse: I can build 25 armaments without touching each other and without encountering any nebulae.  Since you can only have one space station and are unlikely to have more than 5 starships, no player will ever pay extra to build in a nebula.  I don't see any easy way to fix this without seriously shrinking the black or adding an inordinate amount of nebula; even spreading out the pirates doesn't help that much.  Perhaps littering space with satellites that block construction like water?  But this seems inelegant.

The only other strategy I could see to make placement matter is to grant some benefit to armaments placed in mountains and nebulae.

Thoughts?

**Comment by jacobsalmela** [comment]
I realized after the fact how few mountains I added in the smaller map.  But I agree, and never actually did the math as you did, so I appreciate that.

I do like the thought of more challenge.  The satellites seem cool, or maybe a ring of space junk. 

I originally wanted players to have the feeling of building some stuff on the planet, but the excitement came when they went into space, where it was vast and they could venture to other planets.  

Even further down the road, I had envisioned an ASCII Galaxies game, where you explore the entire universe.  

So maybe the game needs more space things, or things that make it harder to get out there.  

The space junk seems interesting and perhaps realistic with how much crap we leave up there.  

Or maybe some sort of ion storm or solar flare could be added to a space-only disaster chart

**Reply by sum1els-com** [reply]
I did think that a small benefit of building in expensive territories might be protection from a disaster or something similar; maybe additional bonus points at the end?

(I have been thinking of a broader ASCII Galaxies game myself, where each player might manage multiple systems...  I'm happy to brainstorm that with you sometime.  =]  )

**Reply by jacobsalmela** [reply]
Awesome!  I suppose we should refine this game a bit more first since we know more what doesn't work.

Yeah, maybe some communication bonus for your ships since they can communicate better with the higher elevation.

**Reply by sum1els-com** [reply]
I have been framing my suggestions as discussions because while I'm excited to improve the game, I also don't want to seem like I'm trying to take it over or anything.  I'm hoping thoughtful conversations and maybe some more playtesting will lead to the right decisions.

**Reply by jacobsalmela** [reply]
The game sat here for two years u touched, so I appreciate it all!  It's why I made it open source in the first place

**Comment by jacobsalmela** [comment]
This should be addressed in https://github.com/jacobsalmela/ascii-planets/releases/tag/v1.2.5. 

Feel free to mark this as an answer for anyone else landing here.
